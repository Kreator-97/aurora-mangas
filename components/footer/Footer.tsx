import Image from 'next/image'

export const Footer = () => {
  return (
    <footer
      className="min-h-[150px] border-t border-solid border-stroke"
    >
      <h2
        className="text-xl md:text-3xl text-center p-4 mt-8">
        SÃ­guenos en nuestras redes sociales
      </h2>
      <div className='flex justify-center gap-8'>
        <Image
          src="/images/social/facebook.svg"
          width={32}
          height={32}
          layout="fixed"
          alt='facebook'
        />
        <Image
          src="/images/social/twitter.svg"
          width={36}
          height={36}
          layout="fixed"
          alt='twitter'
        />
      </div>
    </footer>
  )
}
