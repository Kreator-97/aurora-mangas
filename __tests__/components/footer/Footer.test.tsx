import { render } from '@testing-library/react'
import { Footer } from '../../../components/footer/Footer'

describe('tests on footer component', () => {
  test('should to match with snapshot', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
