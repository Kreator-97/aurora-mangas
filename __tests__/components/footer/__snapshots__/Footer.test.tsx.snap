// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on footer component should to match with snapshot 1`] = `
<div>
  <footer
    class="min-h-[150px] border-t border-solid border-stroke"
  >
    <h2
      class="text-xl md:text-3xl text-center p-4 mt-8"
    >
      SÃ­guenos en nuestras redes sociales
    </h2>
    <div
      class="flex justify-center gap-8"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 32px; height: 32px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
      >
        <img
          alt="facebook"
          data-nimg="fixed"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; width: 36px; height: 36px; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
      >
        <img
          alt="twitter"
          data-nimg="fixed"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
    </div>
  </footer>
</div>
`;
